lib/components/EmbeddableFooter.vue

```diff
@@ -12,173 +16,183 @@
-  import Brand from './Brand'
-  import SharingOptions from './SharingOptions'
-  import config from '@lib/config'
-
-  import { library, default as Fa } from './Fa'
-
-  /**
-   * EmbeddableFooter
-   */
-  export default {
-    i18n,
-    name: 'EmbeddableFooter',
-    beforeMount() {
-      library.add(faShareAlt)
+import { faShareAlt } from "@fortawesome/free-solid-svg-icons/faShareAlt";
+
+import i18n from "@lib/i18n";
+import IframeResizer from "@lib/utils/iframe-resizer";
+import Brand from "./Brand";
+import SharingOptions from "./SharingOptions";
+import config from "@lib/config";
+
+import { library, default as Fa } from "./Fa";
+
+/**
+ * EmbeddableFooter
+ */
+export default {
+  i18n,
+  name: "EmbeddableFooter",
+  beforeMount() {
+    library.add(faShareAlt);
+  },
+  components: {
+    Fa,
+    SharingOptions,
+    Brand,
+  },
+  props: {
+    /**
+     * Title to display next to ICIJ logo.
+     */
+    title: {
+      type: String,
+      default: () => config.get("project.name"),
     },
-    components: {
-      Fa,
-      SharingOptions,
-      Brand
+    /**
+     * Lead sentence to display next to the title.
+     */
+    lead: {
+      type: String,
+      default: "",
     },
-    props: {
-      /**
-       * Title to display next to ICIJ logo.
-       */
-      title: {
-        type: String,
-        default: () => config.get('project.name')
-      },
-      /**
-       * Lead sentence to display next to the title.
-       */
-      lead: {
-        type: String,
-        default: ''
-      },
-      /**
-       * Minimum height for the iframe generated in the embed form.
-       */
-      iframeMinHeight: {
-        type: Number
-      },
-      /**
-       * Minimum width for the iframe generated in the embed form.
-       */
-      iframeMinWidth: {
-        type: Number
-      },
-      /**
-       * Target of the ICIJ logo and title links.
-       */
-      homeUrl: {
-        type: String,
-        default: () => config.get('app.home')
-      },
-      /**
-       * Sharing option values to bind to the sharing-options component on the bottom-right corner.
-       */
-      sharingOptionsValues: {
-        type: Object,
-        default: () => ({})
-      }
+    /**
+     * Minimum height for the iframe generated in the embed form.
+     */
+    iframeMinHeight: {
+      type: Number,
     },
-    data () {
-      return {
-        showShareOptions: false
-      }
+    /**
+     * Minimum width for the iframe generated in the embed form.
+     */
+    iframeMinWidth: {
+      type: Number,
     },
-    mounted () {
-      IframeResizer.create();
-    }
-  }
+    /**
+     * Target of the ICIJ logo and title links.
+     */
+    homeUrl: {
+      type: String,
+      default: () => config.get("app.home"),
+    },
+    /**
+     * Sharing option values to bind to the sharing-options component on the bottom-right corner.
+     */
+    sharingOptionsValues: {
+      type: Object,
+      default: () => ({}),
```

Comment:
```suggestion
      default: () => {},
```

Reply:
It's not the same: 

* `default: () => ({})` returns an empty object
* `default: () => {}` returns null